posthog/cdp/templates/__init__.py

```diff
@@ -42,6 +43,7 @@
     avo,
     gleap,
     google_pubsub,
+    zendesk,
```

Comment:
we should probably sort this list alphabetically... too many conflicts with master on these lines 😅

Reply:
😅